vendor-licenses.txt

```diff
@@ -1142,13 +1142,13 @@ The following NPM packages may be included in this product:
 
 These packages each contain the following license and notice below:
 
-The MIT License (MIT)
-Copyright (c) 2015 Dmitry Ivanov
-
-Permission is hereby granted, free of charge, to any person obtaining a copy of this software and associated documentation files (the "Software"), to deal in the Software without restriction, including without limitation the rights to use, copy, modify, merge, publish, distribute, sublicense, and/or sell copies of the Software, and to permit persons to whom the Software is furnished to do so, subject to the following conditions:
-
-The above copyright notice and this permission notice shall be included in all copies or substantial portions of the Software.
-
+The MIT License (MIT)
```

Comment:
@JanneKiiskila Why are we removing and adding the license copyright text again ?

Reply:
It's not being removed and added, it's just changing the varying line endings from CR/LF to LF (Linux endings).
Issue that originally should not have been happening, if we would have had a .gitattributes in place. (Personally, I think GitHub should not even create fully empty repos, it should always populate a .gitattributes and .gitignore).